web_widget_x2many_2d_matrix/static/src/components/x2many_2d_matrix_field/x2many_2d_matrix_field.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="web_widget_x2many_2d_matrix.X2Many2DMatrixField" owl="1">
+        <div class="table-responsive">
+            <X2Many2DMatrixRenderer
+                list="list"
+                matrixFields="props.matrixFields"
+                showRowTotals="props.showRowTotals"
+                showColumnTotals="props.showColumnTotals"
+                setDirty="props.setDirty"
```

Comment:
```suggestion
                (isDirty) => this.setDirty(isDirty)
```
fixed this error
`UncaughtPromiseError > OwlError
Uncaught Promise > Invalid props for component 'X2Many2DMatrixRenderer': 'setDirty' is undefined (should be a function)
OwlError: Invalid props for component 'X2Many2DMatrixRenderer': 'setDirty' is undefined (should be a function)
    at Object.validateProps (http://localhost:8069/web/assets/debug/web.assets_common.js:19239:19) (/web/static/lib/owl/owl.js:3069)
    at X2Many2DMatrixField.template (eval at compile (http://localhost:8069/web/assets/debug/web.assets_common.js:21606:16), <anonymous>:11:13) (/web/static/lib/owl/owl.js:5436)
    at Fiber._render (http://localhost:8069/web/assets/debug/web.assets_common.js:17791:38) (/web/static/lib/owl/owl.js:1621)
    at Fiber.render (http://localhost:8069/web/assets/debug/web.assets_common.js:17783:18) (/web/static/lib/owl/owl.js:1613)
    at ComponentNode.initiateRender (http://localhost:8069/web/assets/debug/web.assets_common.js:18505:23) (/web/static/lib/owl/owl.js:2335)`

Reply:
Thanks! I'll fix it.

Reply:
Should be fixed now

Reply:
@sonhd91 is it ok for you now? Can you approve this PR?